examples/pylab_examples/fonts_demo.py

```diff
@@ -13,7 +13,7 @@
 alignment = {'horizontalalignment': 'center', 'verticalalignment': 'baseline'}
 # Show family options
 
-families = ['serif', 'sans-serif', 'cursive', 'fantasy', 'monospace']
+families = ['serif', 'sans-serif', 'monospace']
```

Comment:
Any particular reason to drop these. While they may not be the most beautiful fonts Travis does include them correctly here http://matplotlib.org/devdocs/examples/pylab_examples/fonts_demo.html